docs/runtimes/application.ipynb

```diff
@@ -135,6 +135,40 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Other API Gateway configurations\n",
+    "\n",
+    "There are also other parameters that can be configured when creating an API gateway:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "application.create_api_gateway(\n",
+    "    # The name of the API gateway\n",
+    "    name=\"my-api-gateway\",\n",
+    "    # Optional path of the API gateway, default value is \"/\". Note that given path should be supported by deployed application\n",
+    "    path=\"/\",\n",
+    "    # Set True to allow direct port access to the application sidecar\n",
+    "    direct_port_access=False,\n",
+    "    # Set True to force SSL redirect, False to disable. Defaults to mlrun.mlconf.force_api_gateway_ssl_redirect()\n",
```

Comment:
" # Set t0 True to force SSL redirect, False to disable. Defaults to mlrun.mlconf.force_api_gateway_ssl_redirect()\n",